README.md

```diff
@@ -1 +1,54 @@
-# newyorktimes
\ No newline at end of file
+# HTML & CSS3 - The New York Times
+
+This is my first project in Microverse, the purpose of it is to replicate an article from the New York Times website using HTML and CSS3 skills,focused on the layout, positioning and floating.
+
+![screenshot](./images/screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+  - Built With HTML and CSS3
+  - No other languages used
+  - Applied the use of Linters for HTML and CSS3
+
+## Live Demo
+
+[Live Demo Link](https://alexjustalex2020.github.io/newyorktimes/)
+
+
+## Getting Started
+
+** There's no need to install anything in order to see this project. By clicking on the live demo link you should be able too see the full website. **
+
+To get a local copy up and running follow these simple example steps:
+
+  - Just clone this repository to your local files - there's no further installation/setup needed
+  - Images may not load properly
```

Comment:
## Improve the README
- [ ] It would be great if you offered more elaborate information as to why this would be so. DO you mind adding this crucial information here?